graph/config/cytoscape/cytoscape.go

```diff
@@ -88,9 +88,11 @@ type EdgeData struct {
 	Target string `json:"target"` // child node ID
 
 	// App Fields (not required by Cytoscape)
-	Traffic      ProtocolTraffic `json:"traffic,omitempty"`      // traffic rates for the edge protocol
-	ResponseTime string          `json:"responseTime,omitempty"` // in millis
-	IsMTLS       string          `json:"isMTLS,omitempty"`       // set to the percentage of traffic using a mutual TLS connection
+	Traffic      	ProtocolTraffic `json:"traffic,omitempty"`      // traffic rates for the edge protocol
+	ResponseTime 	string          `json:"responseTime,omitempty"` // in millis
+	IsMTLS       	string          `json:"isMTLS,omitempty"`       // set to the percentage of traffic using a mutual TLS connection
+	DestPrincipal 	string		 	`json:"destPrincipal,omitempty"`
+	SourcePrincipal string       	`json:"sourcePrincipal,omitempty"`
```

Comment:
Just needs format a short comments for each new field.  For bonus points alphabetize below // App Fields.